assets/thanos-querier/pod-disruption-budget.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  labels:
+    app.kubernetes.io/component: query-layer
+    app.kubernetes.io/instance: thanos-querier
+    app.kubernetes.io/name: thanos-query
+    app.kubernetes.io/version: 0.20.2
+  name: thanos-querier-pdb
+  namespace: openshift-monitoring
+spec:
+  labelSelector:
```

Comment:
From the CI must-gather we get that the PDB isn't able to find the thanos-querier pods and we can also see that it doesn't have any selector:
```
- apiVersion: policy/v1
  kind: PodDisruptionBudget                                                                                                                                                                                          
  metadata:
    creationTimestamp: "2021-06-09T10:24:02Z"
    generation: 1
    labels:
      app.kubernetes.io/component: query-layer
      app.kubernetes.io/instance: thanos-querier
      app.kubernetes.io/name: thanos-query
      app.kubernetes.io/version: 0.20.2
    name: thanos-querier-pdb
    namespace: openshift-monitoring
    resourceVersion: "47099"
    uid: c35f9eba-0950-48bf-a3e4-fe54901dbf1f
  spec:
    minAvailable: 1
  status:
    conditions:
    - lastTransitionTime: "2021-06-09T10:24:02Z"
      message: ""
      observedGeneration: 1
      reason: InsufficientPods
      status: "False"
      type: DisruptionAllowed
    currentHealthy: 0
    desiredHealthy: 1
    disruptionsAllowed: 0
    expectedPods: 0
    observedGeneration: 1
```

```suggestion
  selector:
```
This should fix the CI failures since the pod disruption budget will now be able to find the thanos-querier pods.